L1Trigger/TrackFindingTMTT/src/KFbase.cc

```diff
@@ -811,9 +813,9 @@ namespace tmtt {
       kfEtaReg = iEtaReg - numEtaRegions_ / 2;
     }
 
-    bool ambiguous = ambiguityMap[kfEtaReg][kfLayer];
-
-    //bool ambiguous = false;
+    bool ambiguous = false;
+    if (Settings_->KFUseMaybeLayers()) ambiguous = ambiguityMap[kfEtaReg][kfLayer];
```

Comment:
Code doesn't compile. Please check this before any PR. e.g. Variable "settings_" should start with lower case "s".

Reply:
My apologies for my bad practice. I fixed the bug and tested the performance of the hybrid tracking using the latest ttbar samples under TrackFindingTMTT/data/ 
![L1TK_elec-pu0_eff_eta](https://user-images.githubusercontent.com/43122048/99634054-2dd2df00-2a40-11eb-8b13-23d39951c436.png)